containers/podman-desktop
# feat: introduce a new section for extensions in the navbar

### What does this PR do?
Introduce a new page to manage extensions. Everything is centralized.

After that, pages settings/extensions,  dashboard/featured will be removed
current page of podman-desktop:extension/extId will be removed and user will get the extension details page



### Screenshot / video of UI



### What issues does this PR fix or reference?

fixes https://github.com/containers/podman-desktop/issues/6083
fixes https://github.com/containers/podman-desktop/issues/3720
fixes https://github.com/containers/podman-desktop/issues/4332
fixes https://github.com/containers/podman-desktop/issues/2677
fixes https://github.com/containers/podman-desktop/issues/6084

### How to test this PR?

Use puzzle icon in the left navbar
should be able to install both Podman Desktop and Docker Desktop extensions (using manual install in top right corner from extensions page)

not yet 100% of coverage but tests are being added
